.github/workflows/publish-containers.yml

```diff
@@ -26,12 +26,21 @@ jobs:
       # Pre-releases, alphas, etc. as :snapshot
       - name: Output image tags from git tag events
         if: ${{ startsWith(github.ref, 'refs/tags/') }}
+        env:
+          GH_TOKEN: ${{ github.token }}
         run: |
-          # Remove everything else than the tagged version
-          DASHBOARD_VERSION=${GITHUB_REF#refs/tags/}
+          LATEST_VERSION=$(gh api repos/saleor/saleor-dashboard/releases/latest | jq .tag_name | xargs)
+          CURRENT_TAG=${GITHUB_REF#refs/tags/}
+
+          if [ "$LATEST_VERSION" = "$CURRENT_TAG" ]; then
+            VERSION_TAG=latest
+          else
+            VERSION_TAG=$CURRENT_TAG
+          fi
+
           echo "
-          DASHBOARD_VERSION=${DASHBOARD_VERSION}
-          CONTAINER_TAGS=${{ steps.image.outputs.image_name }}:${DASHBOARD_VERSION}
+          DASHBOARD_VERSION=${CURRENT_TAG}
+          CONTAINER_TAGS=${{ steps.image.outputs.image_name }}:${VERSION_TAG}
```

Comment:
If 1.2.3 is latest, we need to push:
- `:latest`
- `:1.2.3`

Currently it only pushes `:latest`